tests/analysis/damages/test_damages_utils.py

```diff
@@ -80,3 +87,9 @@ def test_create_summary_statistics(self):
 
         # 3. Verify final expectations
         assert isinstance(result_dict, dict)
+        if all(np.isnan(lanes)):
+            assert np.isnan(result_dict["name1"])
+            assert np.isnan(result_dict["name2"])
+        else:
+            assert result_dict["name1"] == lanes[0]
+            assert result_dict["name2"] == lanes[1]
```

Comment:
these are two separate tests, with nans / without nans. otherwise add as a second pytest parameter the expected results.

Reply:
Reworked